### ISSUE TITLE
Nested arrays

### ISSUE THREAD

**Issue by lmeyer1** [AbcAeffchen/doxygen-php-filters#2]
The filter attribute_short_array_syntax.php does not take into account the possibility for twodimensional or nested arrays.

**Comment by AbcAeffchen** [comment]
This is right. But it seems that only the outer array has to be written in old syntax, to get doxygen work right.

Did you have an example that does not work correctly?
